helix-term/src/commands.rs

```diff
@@ -298,6 +298,8 @@ impl MappableCommand {
         extend_line, "Select current line, if already selected, extend to another line based on the anchor",
         extend_line_below, "Select current line, if already selected, extend to next line",
         extend_line_above, "Select current line, if already selected, extend to previous line",
+        select_line_up, "Select current line, if already selected, extend or shrink line above based on the anchor",
+        select_line_down, "Select current line, if already selected, extend or shrink line below based on the anchor",
```

Comment:
I agree, In fact I don't know why I didn't call it above/below especially since I used these worlds in the description to begin with. Renamed them now